### ISSUE TITLE
Insert or update Chinese character on Microsoft SQL Server will return ????

### ISSUE THREAD

**Issue by gumplei** [laravel/framework#15939]
- Laravel Version: 5..2.45
- PHP Version: 7.0.8
- Database Driver & Version: sqlsrv 
### Description:

I am using Ubuntu and FreeTDS to connect SQL Server 2016. It can correct read the Chinese character from SQL Server but while I tried to insert or update data with Chinese characters. All Chinese characters become ????. It seems that the method prepareBindings in Illuminate\Database\Connection doesn't convert the characters correctly.

I tried to fix this problem by adding the method prepareBindings in Illuminate\Database\SqlServerConnection

```
    public function prepareBindings(array $bindings)
    {
        $grammar = $this->getQueryGrammar();

        foreach ($bindings as $key => $value) {
            // We need to transform all instances of DateTimeInterface into the actual
            // date string. Each query grammar maintains its own date string format
            // so we'll just ask the grammar for the format to get from the date.
            if ($value instanceof DateTimeInterface) {
                $bindings[$key] = $value->format($grammar->getDateFormat());
            } elseif ($value === false) {
                $bindings[$key] = 0;
            } elseif (is_string($value) &&
                mb_detect_encoding($value, mb_detect_order(), true) === 'UTF-8') {
                $bindings[$key] = mb_convert_encoding($value, 'UCS-2LE',
                    'UTF-8');
            }
        }

```

Does anyone know the correct way to fix this problem?
### Steps To Reproduce:

```
App\User::create['chinese_name'=>'陳大文'];
$user = App\User::first();
```

The chinese_name will be ???

**Comment by themsaid** [comment]
@gumplei any progress here? did you get the chance to find a solution for your problem or the possible reason it exists?

**Comment by gumplei** [comment]
@themsaid First, I compiled and installed the latest FreeTDS by myself but it can't fix all Chinese characters issue. In some situations, the new driver still can't convert the Chinese characters correctly. 

Finally, I found out that Microsoft released the ODBC driver for Ubuntu and I try it. Lucky, it help to fix this problem. I installed the latest Microsoft ODBC driver for Linux and compiled the pdo_odbc driver on my machine. than config the laravel to work with pdo_odbc and it seems that cab fixed the Chinese characters problem.

As Laravel looks for the PDO driver by calling PDO::getAvailableDrivers, I need to remove pdo_dblib first to make sure Laravel can find the pdo_odbc driver and I need to update the config/database.php to add my odbc connection setting.
